syntaxes/elixir.json

```diff
@@ -19,6 +19,12 @@
       "match": "([A-Z]\\w*\\s*(\\.)\\s*)*[A-Z]\\w*",
       "name": "entity.name.type.module.elixir"
     },
+    {
+      "begin": "@(spec|type|typep)\\s",
```

Comment:
@jeroenvisser101 is right. 
Probably here is better to match only the `@spec` annotation:

```
 "begin": "@spec\\s",
```